import-service/serverless.ts

```diff
@@ -29,15 +29,16 @@ const serverlessConfiguration: Serverless = {
     },
     environment: {
       AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+      SQS_URL: '${cf:product-service-${self:provider.stage}.SQSQueueUrl}'
     },
     iamRoleStatements: [
       { Effect: 'Allow',
         Action: 's3:ListBucket',
         Resource: [`arn:aws:s3:::${BUCKET}`]
       },
       { Effect: 'Allow',
-        Action: ['s3:*'],
-        Resource: [`arn:aws:s3:::${BUCKET}/*`]
+        Action: 'sqs:*',
```

Comment:
Shouldn't the import-service still have _write_ access to the bucket, to be able to move the file to /parsed?

Reply:
it has write access as all my files end up in the parsed folder